tests/e2e/commands/subnet.go

```diff
@@ -463,6 +463,52 @@ func SimulateFujiRemoveValidator(
 	return string(output)
 }
 
+func SimulateFujiTransformSubnet(
+	subnetName string,
+	key string,
+) (string, error) {
+	// Check config exists
+	exists, err := utils.SubnetConfigExists(subnetName)
+	gomega.Expect(err).Should(gomega.BeNil())
+	gomega.Expect(exists).Should(gomega.BeTrue())
+
+	// enable simulation of public network execution paths on a local network
+	err = os.Setenv(constants.SimulatePublicNetwork, "true")
+	gomega.Expect(err).Should(gomega.BeNil())
+	cmd := exec.Command(
+		CLIBinary,
+		SubnetCmd,
+		ElasticTransformCmd,
+		"--fuji",
+		"--key",
+		key,
+		"--tokenName",
+		"BLIZZARD",
+		"--tokenSymbol",
+		"BRRR",
+		"--denomination",
+		"0",
+		"--default",
+		"--force",
+		subnetName,
+	)
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		fmt.Println(cmd.String())
+		fmt.Println(string(output))
+		utils.PrintStdErr(err)
+		err2 := os.Unsetenv(constants.SimulatePublicNetwork)
```

Comment:
if error is checked as mentioned earlier, there is no need to do this unset and also to do the return, just copy the pattern as in other commands (check for err != nil, print all info, then aftee the if, do the gomega.Expect)